### PR TITLE
Create like comment

### PR DESCRIPTION
**Pull request by cyburns**

### CODE REVIEW

File: app/_layout.tsx [hunk 1]
```diff
@@ -125,7 +123,7 @@ function RootLayoutNav() {
   return (
     <NavigationContainer independent={true}>
       <Stack.Navigator initialRouteName="welcome">
-        {!isUser && !isFullyRegistered ? (
```
Comment (avwilliams1995) [inline app/_layout.tsx:128]: So login will now be required?

File: app/screens/create-post/selectedPost.tsx [hunk 2]
```diff
@@ -1,77 +1,91 @@
...
 
 interface SelectedProps {
-  data: any;
+  data: {
+    image: string;
+    firstLine: string;
+    secondLine: string;
+  };
   resetSearch: () => void;
+  setSongName: Dispatch<SetStateAction<string>>;
```
Comment (avwilliams1995) [inline app/screens/create-post/selectedPost.tsx:14]: Why are we saving the current song selection to redux? Are we needing to access that info elsewhere besides in the selectedpost component? The SelectedPost can encapsulate the data needed for the selected song to be able to play. We just need to be able to save this selected data to firebase with the post

File: app/screens/create-post/selectedPost.tsx [hunk 3]
```diff
@@ -1,77 +1,91 @@
...
+  useEffect(() => {
+    setSongName(data.firstLine);
+    setArtistName(data.secondLine);
+    setAlbumCover(data.image);
+  });
+
   return (
     <View style={styles.container}>
-      <AntDesign
-        name="close"
```
Comment (avwilliams1995) [inline app/screens/create-post/selectedPost.tsx:15]: We need a close button. What if the user changes their mind? Needs to be able to reset

File: app/screens/create-post/selectedPost.tsx (hunk 2 above)
Comment (cyburns) [reply app/screens/create-post/selectedPost.tsx:14]: this is calebs or your work not mine

File: app/screens/create-post/selectedPost.tsx (hunk 3 above)
Comment (cyburns) [reply app/screens/create-post/selectedPost.tsx:15]: if you look i just moved it it still there

File: app/screens/create-post/selectedPost.tsx [hunk 4]
```diff
@@ -1,77 +1,91 @@
...
       <Image source={{ uri: data.image || "" }} style={styles.image} />
-      <View style={styles.infoBox}>
+      <View style={{ flex: 1, flexDirection: "column" }}>
         <Text
           style={{
-            color: Colors.white,
             fontFamily: "mon-b",
-            fontSize: 20,
-            marginTop: 5,
+            fontSize: 24,
```
[the remark this answers is not in the document]
Comment (cyburns) [reply app/screens/create-post/selectedPost.tsx:39]: ideally we would have animated text like Spotify carousel text

**Review by avwilliams1995 — APPROVED** [review]

**Review by avwilliams1995 — APPROVED** [review]

**Pull request merged**